src/services/authService.ts

```diff
@@ -5,26 +5,41 @@
  *
  */
 import {
-  AuthError,
+  // AuthError,
   OAuthResponse,
   User,
-  UserResponse,
+  //  UserResponse,
 } from '@supabase/supabase-js';
 import { supabaseClient } from './supabaseClient';
 
+//////////////
+const user: User = {
```

Comment:
Do we still need all this? It was a quick workaround, but I don't think we use it anymore now that you fixed it.

Reply:
Yes, we can remove this code